src/otx/core/data/dataset/base.py

```diff
@@ -141,11 +142,18 @@ def __getitem__(self, index: int) -> T_OTXDataEntity:
         msg = f"Reach the maximum refetch number ({self.max_refetch})"
         raise RuntimeError(msg)
 
-    def _get_img_data_and_shape(self, img: Image) -> tuple[np.ndarray, tuple[int, int]]:
+    def _get_img_data_and_shape(
+        self,
+        img: Image,
+        roi: dict[str, Any] | None = None,
+    ) -> tuple[np.ndarray, tuple[int, int], dict[str, Any] | None]:
```

Comment:
Could you add docstring?

Reply:
Done https://github.com/openvinotoolkit/training_extensions/pull/4105/commits/f703e596013858fbfc9cc587f25a9b4579f50f9b